src/carousel/MagnifyHint.js

```diff
@@ -81,6 +111,10 @@ const styles = theme => ({
 
 const useStyles = makeStyles(styles, { name: 'RSFMagnifyHint' })
 
+/**
+ * An element overlaid on a [`Carousel`](/apiReference/carousel%f2Carousel) that displays a tip for a
+ * user to hover/click the Carousel in order to zoom in.
+ */
```

Comment:
Same here, after propTypes

Reply:
I agree with doing this for `CarouselThumbnails` since that one is just hanging out in the middle of the file, but this one is actually like most of the other components in the app. If we want to change all of these to always move `export default Xxxx` to the end, I'm okay with that – but I think it would be best done in a separate PR since there are dozens of cases.